ue4docker/dockerfiles/ue4-build-prerequisites/windows/install-prerequisites.bat

```diff
@@ -1,7 +1,11 @@
 @rem Install the chocolatey packages we need
 choco install -y git --params "'/GitOnlyOnPath /NoAutoCrlf /WindowsTerminal /NoShellIntegration /NoCredentialManager'" || goto :error
-@rem pdbcopy.exe from Windows SDK is needed for creating an Installed Build of the Engine
-choco install -y choco-cleaner curl vcredist-all windows-sdk-10-version-1809-windbg || goto :error
+
+@rem We're installing Windows SDK here instead of through VS installer for several reasons:
+@rem 1. pdbcopy.exe is needed for creating an Installed Build of the Engine and it isn't installed through VS
+@rem 2. UE-4.27 failts to compile against Windows SDK older than 18362 and 18362 isn't available through VS2017 installer
+choco install -y choco-cleaner curl vcredist-all windows-sdk-10.1 || goto :error
```

Comment:
This is a floating version target. Would it be better to explicitly use [`windows-sdk-10-version-1903-all`](https://community.chocolatey.org/packages/windows-sdk-10-version-1903-all), which is currently the same version but passes `/Features +` to the installer, perhaps it installs more stuff?). That version won't change if the [`windows-sdk-10.1`](https://community.chocolatey.org/packages/windows-sdk-10.1) maintainer decides to update to a new version again.

That said, if the version floats forward, that shouldn't be a problem for us, new SDKs can generally target older versions, and I doubt UE4 is using _any_ runtime support that's even Windows 10-specific on the Windows target. And it still points at the 2019 SDK release, so perhaps it won't float forward (or is abandoned).

Reply:
> the Windows Server 2022-paired SDK release is "2104"

![WAT](https://www.pvsm.ru/images/Scala-WAT-kollekcii.jpg)